hydra-core/lib/tasks/hydra.rake

```diff
@@ -4,7 +4,8 @@ namespace :hydra do
   desc "Start a solr, fedora and rails instance"
   task :server do
     with_server('development') do
-      IO.popen('rails server') do |io|
+      # If HOST specified, bind to that IP with -b
+      IO.popen("rails server #{%Q[-b #{ENV['HOST']}] if ENV['HOST']}") do |io|
```

Comment:
@jcoyne I've corrected this now...and did a rebase squash.  See https://github.com/projecthydra/hydra-head/pull/382/commits/38b75222e2e6885c63a8847e9ce04635f35fa30e